aws_lambda_powertools/utilities/data_classes/dynamo_db_stream_event.py

```diff
@@ -182,8 +182,10 @@ def approximate_creation_date_time(self) -> Optional[int]:
         item = self.get("ApproximateCreationDateTime")
         return None if item is None else int(item)
 
+    # This override breaks the Mapping protocol of DictWrapper, it's left here for backwards compatibility with
+    # a 'type: ignore' comment. This is currently the only subclass of DictWrapper that breaks this protocol.
```

Comment:
```suggestion
    # a 'type: ignore' comment. See #1516 for discussion
```

This is true atm of writing but may not be. Minor update to point to PR instead if one needs more info.